### PR TITLE
Remove adlists tab from settings page

### PR DESCRIPTION
**Pull request by yubiuser**
**By submitting this pull request, I confirm the following:** `{please fill any appropriate checkboxes, e.g: [X]}`

`{Please ensure that your pull request is for the 'devel' branch!}`

- [x] I have read and understood the [contributors guide](https://github.com/pi-hole/AdminLTE/blob/master/CONTRIBUTING.md), as well as this entire template.
- [x] I have made only one major change in my proposed changes.
- [x] I have commented my proposed changes within the code.
- [ ] I have tested my proposed changes.
- [x] I am willing to help maintain this change if there are issues with it later.
- [x] I give this submission freely and claim no ownership.
- [x] It is compatible with the [EUPL 1.2 license](https://opensource.org/licenses/EUPL-1.1)
- [x] I have squashed any insignificant commits. ([`git rebase`](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html))
- [x] I have Signed Off all commits. ([`git commit --signoff`](https://git-scm.com/docs/git-commit#Documentation/git-commit.txt---signoff))

---

**What does this PR aim to accomplish?:**

Removes the obsolete `Adlists` tab from the settings page. Since v5.0 (which was released over a year ago) it was only a placeholder for `Group Management/Adlists`-

### CODE REVIEW

**Comment by yubiuser** [comment]
Runs locally without any complaints

```
chrko@ThinkPad-X230:~/Software/pihole_repositories/AdminLTE$ npm test

> AdminLTE@1.0.0 test
> npm run prettier:check && npm run xo


> AdminLTE@1.0.0 prettier:check
> prettier -l "style/*.css" "style/themes/*.css" "scripts/pi-hole/**/*.js"

scripts/pi-hole/js/settings.js
chrko@ThinkPad-X230:~/Software/pihole_repositories/AdminLTE$ 
```

**Comment by DL6ER** [comment]
> scripts/pi-hole/js/settings.js

That's the complaint. It won't tell you more, just the file with the issue...

File: scripts/pi-hole/js/settings.js [hunk 1]
```diff
@@ -121,7 +121,8 @@ $(".confirm-disablelogging-noflush").confirm({
 });
 
 $(".api-token").confirm({
-  text: "Make sure that nobody else can scan this code around you. They will have full access to the API without having to know the password. Note that the generation of the QR code will take some time.",
+  text:
+    "Make sure that nobody else can scan this code around you. They will have full access to the API without having to know the password. Note that the generation of the QR code will take some time.",
```
Comment (DL6ER) [inline scripts/pi-hole/js/settings.js]: It is very likely this change

Comment (yubiuser) [reply scripts/pi-hole/js/settings.js]: Strange... this is was `npm run prettier:fix` introduced.

Thanks.

Comment (DL6ER) [reply scripts/pi-hole/js/settings.js]: Your `prettier` may be outdated, try `npm i` to upgrade it/install the latest version

Comment (yubiuser) [reply scripts/pi-hole/js/settings.js]: Thanks. Fixed

**Review by DL6ER — APPROVED** [review]

**Comment by mrded** [comment]
Here you are! I was looking for "adlists tab". It has moved, make sense.

**Pull request merged**